lib/sqlalchemy/dialects/postgresql/json.py

```diff
@@ -340,4 +364,28 @@ def contained_by(self, other):
                 CONTAINED_BY, other, result_type=sqltypes.Boolean
             )
 
+        def delete_path(self, array):
+            """JSONB expression. Deletes field or array element specified in
+            the argument array.
+            """
+            return self.operate(DELETE_PATH, array, result_type=JSONB)
+
+        def path_exists(self, other):
+            """Boolean expression. Test for presence of item given by the
+            argument JSONPath expression.
+            """
```

Comment:
**mike bayer** (zzzeek) wrote:

these need to have ``..versionadded`` notes

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4336

Reply:
**Federico Caselli** (CaselIT) wrote:

Ack

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4336